code/api/views.py

```diff
@@ -109,9 +110,51 @@ def get(self, request, author_id, post_id):
 
 @method_decorator(csrf_exempt, name='dispatch')
 class PostCommentsView(View):
+    '''
+        HANDLE Comment GET and POST
+    '''
 
     def get(self, request, author_id, post_id):
-        return HttpResponse("This is the authors/aid/posts/pid/comments/ endpoint")
+        # Send all comments 
+        # TODO: Update json response and data serialization to match spec
+        try:
+            comments = Comment.objects.filter(post=post_id).order_by('-pub_date')
+            
+            serializedComments = []
+            for comment in comments:
+                serialized_obj = serializers.serialize('json', [ comment, ])
+                serializedComments.append(serialized_obj)
```

Comment:
As mentioned in the meeting, you can take a look at the work I did for authors and followers for how to correctly format the response according to the spec

Reply:
thanks! create #89  for this